cherrypy/tutorial/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3'
```

Comment:
What about `3.3`? There's useful image cache option supported in recent versions

Reply:
Image cache is a docker feature as far as I know. Version 3 main difference over 2 is support for swarm.

Reply:
There's `cache_from` key in build section, which is not available in old compose versions: https://github.com/docker/compose/pull/4514#issuecomment-305052337